jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -650,7 +636,8 @@ void java_object_factoryt::gen_nondet_pointer_init(
     update_in_placet::NO_UPDATE_IN_PLACE,
     location);
 
-  auto set_null_inst=get_null_assignment(expr, pointer_type);
+  const auto set_null_inst =
+    code_assignt{expr, null_pointer_exprt{pointer_type}, loc};
```

Comment:
I think the style convention we follow is `const code_assignt set_null_inst{expr, null_pointer_exprt{pointer_type}, loc};` :corn: